File: challenges/3-area-trapezio/js/akiratorres/trapeze.js
const args = (process.argv.slice(2));
const largerBase = Number(args[0]);
const smallerBase = Number(args[1]);
const height = Number(args[2]);

function getArea(largerBase, smallerBase, height) {
    area = (largerBase + smallerBase) * height / 2;
    if (isNaN(area) == true) {
        return "Um dos argumentos não era um número";
    }

    return `A área do trapézio é: ${area}`;
}

console.log(getArea(largerBase, smallerBase, height));

File: challenges/3-area-trapezio/dart/Alba-22/trapeze.dart
import 'dart:io';

void main() {
  try {
    stdout.write("Base Maior: ");
    double largerBase = double.parse(stdin.readLineSync());
    stdout.write("Base Menor: ");
    double smallerBase = double.parse(stdin.readLineSync());
    stdout.write("Altura: ");
    double height = double.parse(stdin.readLineSync());
    double area = (largerBase + smallerBase) * height / 2;
    stdout.write("A área do trapézio é: $area");
  }
  catch (e) {
    if (e.runtimeType == FormatException) {
      print("Input Inválido");
    }
    else {
      print("Ocorreu um erro :(");
    }
  }
}